{{ cookiecutter.formal_name }}/{{ cookiecutter.formal_name }}/main.m

```diff
@@ -208,3 +186,61 @@ void crash_dialog(NSString *details) {
     // Show the crash dialog
     [alert runModal];
 }
+
+
+NSString * format_traceback(PyObject *type, PyObject *value, PyObject *traceback) {
+    
+    NSRegularExpression *regex;
+    NSString *traceback_str;
+    PyObject *traceback_list;
+    PyObject *traceback_module;
+    PyObject *format_exception;
+    PyObject *traceback_unicode;
+    
+
+    if (traceback == NULL) {
+        NSLog(@"Could not retrieve traceback");
+        return NULL;
```

Comment:
What will `crash_dialog()` do with this? Seems like we should return the "Could not retrieve traceback" string (or something similar) so that the dialog can at least display why it can't display

Reply:
Good point.